tests_e2e/tests/scripts/agent_persist_firewall-access_wireserver

```diff
@@ -90,15 +90,21 @@ echo "Finally online, Time: $(date --utc +%FT%T.%3NZ)"
 echo "Trying to contact Wireserver as $USER to see if accessible"
 
 echo ""
-echo "IPTables before accessing Wireserver"
-sudo iptables -t security -L -nxv -w
+echo "Firewall configuration before accessing Wireserver:"
+if command -v iptables; then
+  sudo iptables -t security -L -nxv -w
+else
+  sudo nft list table walinuxagent
+fi
 echo ""
 
 WIRE_IP=$(cat /var/lib/waagent/WireServerEndpoint 2>/dev/null || echo '168.63.129.16' | tr -d '[:space:]')
-if command -v wget >/dev/null 2>&1; then
-    wget --tries=3 "http://$WIRE_IP/?comp=versions" --timeout=5 -O "/tmp/wire-versions-$USER.xml"
-else
+if command -v curl >/dev/null 2>&1; then
     curl --retry 3 --retry-delay 5 --connect-timeout 5 "http://$WIRE_IP/?comp=versions" -o "/tmp/wire-versions-$USER.xml"
+elif command -v wget >/dev/null 2>&1; then
+    wget --tries=3 "http://$WIRE_IP/?comp=versions" --timeout=5 -O "/tmp/wire-versions-$USER.xml"else
+else
+    http_get.py "http://168.63.129.16/?comp=versions" --timeout 5 --delay 5 --tries 3
```

Comment:
I created this simple tool by doing surgery on some of the networking code in the agent. I used it to debug the above issue with wget. Since I already created it, I'm leaving it here as a fallback to curl/wget (initially it was meant as a replacement for them, but just changing the order of curl/wget fixed the original issue)